.prettierrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
-  ...require('@netlify/eslint-config-node/.prettierrc.json'),
-  endOfLine: 'auto',
-}
+  useTabs: false,
+  singleQuote: true,
+  trailingComma: 'all',
```

Comment:
It should be `semi: false` too, to match the rest of our code

Reply:
I think just copy https://github.com/netlify/eslint-config-node/blob/main/.prettierrc.json for now